taxi/tests/tests_forms.py

```diff
@@ -0,0 +1,116 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import (DriverCreationForm,
```

Comment:
Place each import on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change at all places